### PR TITLE
Upgrade babel + plugins, use es2020 preset

### PR DESCRIPTION
**Pull request by johallar**
# Merge Checklist
This will allow us to use es2020 features like optional chaining! And performs some babel maintenance so we don't fall too far behind

## :wrench: Issue(s) fixed:
- [ ] Author referenced issue(s) fixed by this PR:
- [ ] Fixes #0

## :smoking: Smoke Test
- [ ] Works in chrome
- [ ] Works in firefox
- [ ] Works in safari
- [ ] Works in ie edge
- [ ] Works in ie 11

## :ship: Merge
- [ ] author crafted PR's title into release-worthy commit message.

### CODE REVIEW

File: src/utils/utils-contour.js [hunk 1]
```diff
@@ -242,6 +239,8 @@ export const getContourScales = ({ strokeWidth, opacity, color }) => {
     }
   ]
 }
+export const isContourType = (state = {}) =>
+  state?.data?.[0]?.type === CONTOUR_TYPE
```
Comment (johallar) [inline src/utils/utils-contour.js:243]: Optional chaining woo!

**Review by DylanTackoor — COMMENTED** [review]
builds+starts locally for me, but there's a merge conflict

**Comment by johallar** [comment]
Merging in master + rebuilding now!

**Comment by johallar** [comment]
@vastcharade any objections here?

**Review by vastcharade — APPROVED** [review]
lgtm. let's go for it

**Pull request merged**